share/ruby-build/graalvm-20.1.0

```diff
@@ -0,0 +1,13 @@
+case $(uname -s) in
+Linux)
+  install_package "graalvm-20.1.0" "https://github.com/graalvm/graalvm-ce-builds/releases/download/vm-20.1.0/graalvm-ce-java8-linux-amd64-20.1.0.tar.gz#4fac212b37cd548831fd6587dd4d59dc068068815aa20323b47fde9529d6bb6e" graalvm
+  ;;
+Darwin)
+  use_homebrew_openssl
+  install_package "graalvm-20.1.0" "https://github.com/graalvm/graalvm-ce-builds/releases/download/vm-20.1.0/graalvm-ce-java8-darwin-amd64-20.1.0.tar.gz#3b9fd8ce84c9162a188fde88907c66990db22af0ff6ae2c04430113253a9a634" graalvm
```

Comment:
How do we choose between the java8 package and the java11 package? When should/would we switch to java11?

Reply:
I'm being pragmatic here, since we are in a Ruby installer, TruffleRuby is currently better tested on java 8 so I picked java 8.
I think having both would be too many variants.
Of course we can change it over time, that's easy since there are separate definition files per GraalVM version.